package.json

```diff
@@ -95,6 +95,7 @@
     "@react-native-picker/picker": "^2.4.2",
     "@react-navigation/compat": "^5.3.20",
     "@react-navigation/core": "^6.2.1",
+    "@react-navigation/devtools": "^6.0.10",
```

Comment:
should this be a dev dependency instead?

Reply:
Since it's using a hook we can't conditionally call it in the `NavigatorWrapper.tsx`. The guide has it as a dependency instead of a dev dependency.